### ISSUE TITLE
Announcements and discussion about project

### ISSUE THREAD

**Issue by KeNaCo** [Michael-F-Bryan/auto-changelog#59]
I created this issue to create a somehow common channel to communicate and handle discussions and announcements about whole project.

**Comment by KeNaCo** [comment]
First I want to apologize for the past few months lack of communication and development.
Series of live events eats all my time and energy to progress with the project. Then corona hits and my employer was hit hard, so again, no spare time and energy.
Month in quarantine, things are settling down, I improved my time management and can again work on the project.

**Comment by LeMimit** [comment]
Hello ! Welcome back !

No problem. You have to focus on your life first and then on annexe project like this.
This is a good news that you are back.

**Comment by KeNaCo** [comment]
But I'm still slow :D

**Comment by Celeborn2BeAlive** [comment]
Thank your for starting this ! I will use it for my new python projects and I may contribute if I find bugs or features that I need.

**Comment by adam-gligor** [comment]
hello, I'd like to use this library. What i'm still missing is the [breaking-change](https://github.com/Michael-F-Bryan/auto-changelog/issues/64) feature, which is in a PR with no recent activity. Do you accept contributions? I'd like to work on this.

**Comment by KeNaCo** [comment]
@adam-gligor hi, I'm opened to any PR. Some work on breaking changes already proposed in #57. If you want to continue or create your own solution, just let me know in issue #64 or PR #57.

**Comment by KeNaCo** [comment]
@LeMimit, @stdtom just to let you know, I see your PRs. I plan to work on the project on sunday.

**Comment by LeMimit** [comment]
Ok cool :) Thanks 👍🏻

**Comment by stdtom** [comment]
I split my PR into several as requested. I also added a few feature requests but I do not have the permission to tag them.
Let me know what you think about them. I would be happy to implement them based on your feedback.

**Comment by stdtom** [comment]
@KeNaCo One further question/proposal:
I know you have cloned this project to GitLab and run a CI/CD pipeline there.
Have you considered unsing Travis CI for CI/CD or any other CI that integrates with GitHub? 
With an GitHub integrated CI/CD pipeline there would the benefit of running the pipeline on every pushed commit and on every PR including the feedback to the commit or checks annoted to the PR.

I added a Travis pipeline line to my fork of this project &ndash; in case you want to have a look on it: https://github.com/stdtom/auto-changelog/pull/8

**Comment by KeNaCo** [comment]
Yes, I was considering Travis, but at the time, they changed policies for the free tier and it was super-slow.
Regarding GitLab, it's more complicated. It is integrated with Github the same way Travis is. But, in this (upstream) repo, I have only maintainer rights. So I have my fork (origin) and this one is automatically mirrored to GitLab where CI runs - for each commit.

I don't like the setup and I'm thinking for some time to ask Mr. Brian if I can migrate upstream to one of my repositories.

**Comment by KeNaCo** [comment]
During the vacation, I'm working on improving tests and manual comparison against the JS version.

**Comment by LeMimit** [comment]
@stdtom @KeNaCo For the integration of a new pipeline [Github actions](https://github.com/features/actions) can be a solid choice here. I been using it in my projects, it is quite easy to put in place, quick.

**Comment by KeNaCo** [comment]
@LeMimit I checked it and it's the same thing as Gitlab CICD (and any other CICD tool in this manner). Different names, different syntax but it's the same. 

The open question still is: Should I migrate the project under some of my repositories? WDYT (@LeMimit , @stdtom)? Also what @Michael-F-Bryan think about this idea, as this is his repository and I will like to honor his will.

**Comment by KeNaCo** [comment]
Side-topic. During testing the JS version, I find out interesting behavior: https://github.com/cookpete/auto-changelog/issues/193

**Comment by Michael-F-Bryan** [comment]
> Should I migrate the project under some of my repositories?

I'm happy to transfer the project's ownership to you.

**Comment by LeMimit** [comment]
I do not know the status of @Michael-F-Bryan about this projet. If he is not going to contribute anymore. I feel like transfering the ownership to you @KeNaCo seems right. It will give you more rights to be able to make this cool projet even better. 
If I remember well the issues and commit history will be perserved.

Adding a CI pipeline is for me mandatory to in this kind of project.

**Comment by Michael-F-Bryan** [comment]
@KeNaCo you will need to delete your (KeNaCo/auto-changelog) before I can transfer the repository.

> I do not know the status of @Michael-F-Bryan about this projet. If he is not going to contribute anymore. 

@LeMimit, see #11.

> If I remember well the issues and commit history will be perserved.

That is correct. All history of the entire project is preserved when it is transferred. GitHub also wire up redirects so people trying to use the old URL still go to the right place. I believe it'll also tell you to the repo was transferred and that you should update any old links or `git` remotes.

**Comment by KeNaCo** [comment]
~~Ok, I will delete my repo.~~
@Michael-F-Bryan GitHub and GitLab repost deleted.

**Comment by Michael-F-Bryan** [comment]
@KeNaCo the transfer has been initiated.

**Comment by KeNaCo** [comment]
@Michael-F-Bryan you will kill me. :facepalm:
I learned new things about gitlab. Transfer request have expiration. And it was not visible in web UI, only think I find is mail.

So my bad, my appologizes and next time I will be more careful of checking the mail.

**Comment by Michael-F-Bryan** [comment]
@KeNaCo you should be able to accept the transfer now.

**Comment by KeNaCo** [comment]
Success :smile:

**Comment by gsemet** [comment]
hi. is this project still active? I see a bunch of merge request that can be great, do you need help to integrate and release?

**Comment by KeNaCo** [comment]
Hi,
thank you for your comment. 
Yes, the project is still active. I just don't have time on it right now (I'm working for a company in the traveling industry, so rough times and work overclock), plus my ISP sucks so even during Sunday evenings (my usual free time) my internet is not usable - this will improve soon (change of ISP).

So as soon as it will be possible, I will sit down, setup CICD (the reason we transferred the project), and go thru PRs and Issues..

**Comment by televi** [comment]
@KeNaCo - any luck getting online?  This package is essentially broken at this point due to changes in Markupsafe being pulled in by an old (unsupported) version of jinja2 via ^2.10 in pyproject.toml.
